e2e/process-services/pages/tasks-list.page.ts

```diff
@@ -46,6 +46,7 @@ export class TasksListPage {
 
     async selectRow(taskName: string): Promise<void> {
         await this.dataTable.selectRow('Name', taskName);
+        await browser.sleep(1000);
```

Comment:
there's no way to avoid browser.sleep ?

Reply:
it needs modify on the demo page not the best but for now nope